code/modules/cargo/packs.dm

```diff
@@ -2182,6 +2219,55 @@
 	crate_name = "art supply crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 25000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
+	name = "Captain Headset Key"
+	desc = "A spare Captain headset key, just in case The Captain ''lost'' it again." 
+	access = ACCESS_CAPTAIN
+	cost = 20000
+	contains = list(/obj/item/encryptionkey/heads/captain)
+	crate_name = "captain headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cargo_headsets
```

Comment:
These would be better as a department protolathe thing instead of cargo pack bloat, if anything.

Reply:
Ok fair